oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/metrics/expression/EqualMatch.java

```diff
@@ -23,6 +23,19 @@
 
 @FilterMatcher("stringMatch")
 public class EqualMatch {
+
+    public boolean match(String left, String right) {
```

Comment:
`equal match` support literal string?

Reply:
Yes, passed the unit test.

https://github.com/apache/skywalking/blob/720c1dd92f3883fc8841b0dbbb3508b40a122168/oap-server/oal-rt/src/test/java/org/apache/skywalking/oal/rt/parser/ScriptParserTest.java#L87-L111

Reply:
I noticed there are tests, just confirm.